### ISSUE TITLE
Any way to remove the mdc-classes?

### ISSUE THREAD

**Issue by marcusjm** [SamProf/MatBlazor#819]
**Describe the question**
A clear and concise description of what the question is.

I am trying to implement tailwind classes with the functionality of the datatable etc. The problem is that the mdc class seems hard coded. Can it be removed with some override?

**Comment by Christian-Oleson** [comment]
@marcusjm ,
the MDC classes are specific to Material Design, which is the basis of the MatBlazor components. This, in my opinion, would not make sense to be able to remove.

**Comment by tesar-tech** [comment]
@Christian-Oleson  I started using MatBlazor yesterday, not because of material design, but **because of components it offers**. It does make sense to have own style on components. (Or I misunderstood the whole point of this library(?)). 

For example, I would like to plug in [bootstrap](https://getbootstrap.com/) or [Tailwind](https://tailwindcss.com/) libraries. Now the official [templating](https://www.matblazor.com/Themes) offers just primary and secondary color change.

**Comment by CSOleson** [comment]
> @Christian-Oleson I started using MatBlazor yesterday, not because of material design, but **because of components it offers**. It does make sense to have own style on components. (Or I misunderstood the whole point of this library(?)).
> 
> For example, I would like to plug in [bootstrap](https://getbootstrap.com/) or [Tailwind](https://tailwindcss.com/) libraries. Now the official [templating](https://www.matblazor.com/Themes) offers just primary and secondary color change.

Got it. At this point, it is not something that is supported. From my understanding, the only intention was to support material design (thus the name of MatBlazor, standing for Material Blazor). Bootstrap is 100% an opposing design choice, so at this point it would not be supported. If the support of tailwinds requires the MDC removal, then that wouldn’t be supported either.


We can definitely have you open a feature request to get that added, I’m just not certain that the intention ever was to support anything aside from Material Design. Hope this helps @tesar-tech

**Comment by marcusjm** [comment]
Yeah I would like that. I only use Taliwind for everything nowadays.

**Comment by Christian-Oleson** [comment]
@marcusjm ,
Is your question answered?

**Comment by marcusjm** [comment]
> @marcusjm ,
> Is your question answered?

Yes

**Comment by Christian-Oleson** [comment]
@marcusjm ,
Awesome. Would you mind closing this issue then? Likewise, if still desired, you could write up a feature request for supporting other designs like bootstrap, tailwinds, etc.
